dw-dialog.js

```diff
@@ -207,6 +197,8 @@ export const DwModalDialogMixin = (baseElement) => class DwModalDialog extends b
       super.open(triggerEl)
       return;
     }
+    this.resizeObserver.observe(this._dialogContentEl);
```

Comment:
@hitendreamworld for better logic, following updates are needed:

1. Don't write custom getter of `_dialogContentEl` 
2. Instead, during `observe` set this instance variable, using the same logic written for the custom getter.
3. Use the same instance variable during `unobserve`.